PredefinedDynamicalSystems jacobian fixes

rusandris opened the pull request:
Fixes for problems discussed in [ Update Systems submodule to v3 #7 ](https://github.com/JuliaDynamics/PredefinedDynamicalSystems.jl/issues/7) .

Datseris commented:
While this is correct, it is inefficient to create a matrix and then port it to an `SMatrix`. You should re-write the matrix by transposing the arguments and putting commas everywhere.

(transpose arguments: matrices in Julia are column major so you go first by columns and then by rows. hence, if you wanted to input each element of the matrix one by one to the constructor, which is the most efficient way, you firstr go all elements of first column then second column etc. Instead of going row by row which is how you make a matrtix with the space and `;` syntax)

rusandris commented:
I've changed (hopefully) everything to the more efficient element-by-element `SMatrix` constructor. Indeed, I think it would make a difference if the jacobian is a called many times. I also noticed that in some cases (`roessler` and `chua`), the element-by-element (elements separated by commas) constructor was used originally, but incorrectly (going row-by-row instead of column-by-column), so I also corrected that.

Datseris reviewed (approved):
Thank you very much. As a final change, please increment minor version (second number) in Project.toml so that I can tag a release after merging.
